src/client/dfuse/il/intercept.h

```diff
@@ -28,6 +28,8 @@
  * fcntl (for now though we likely need for dup)
  */
 #define FOREACH_ALIASED_INTERCEPT(ACTION)                                                          \
+	_Pragma("GCC diagnostic push")								   \
+	_Pragma("GCC diagnostic ignored \"-Wmissing-attributes\"")				   \
```

Comment:
I think we've seen something along these lines before, what's the actual issue?

Reply:
Yeah it was complaining about `fseek64()` at https://codebrowser.dev/glibc/glibc/libio/stdio.h.html#780, where it has `__nonnull((1))` attribute but ours doesn't have it.

BTW, I have gcc-13 installed as well, and it will hit the same issue; not sure why it can compile on Fedora . I'm using a Debian based system.